moboff.py

```diff
@@ -159,10 +159,10 @@ def download(link, newdevice, video, delete,send):
     if send:
         for item in pb.chats:
             print(pb.chats.index(item), item, "\n")
-            index=rawinput("Enter the corresponding chat no. for the person you want to send the file to.")
+        index=rawinput("Enter the corresponding chat no. for the person you want to send the file to.")
```

Comment:
Add a space after "end the file to. ") so that the entered value is separated from this text.